File: src/main/java/io/teipub/HomeController.java
package io.teipub;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.servlet.ModelAndView;

/**
 * Created by tei on 2016-08-15.
 * teipub.io
 */
@Controller
public class HomeController {

    @RequestMapping("/")
    public ModelAndView home() {
        ModelAndView mav = new ModelAndView("home");
        mav.addObject("name", "전경민");
        return mav;
    }
}

File: src/main/kotlin/teipub/HomeController.kt
package teipub

import org.springframework.stereotype.Controller
import org.springframework.ui.Model
import org.springframework.web.bind.annotation.RequestMapping

/**
 * Created by tei on 2016-08-15.
 * teipub.io
 */
@Controller
class HomeController {
    @RequestMapping("/")
    fun home(model: Model): String {
        model.addAttribute("name", "teipub")
        return "home"
    }
}


